src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -83,5 +109,15 @@ limitations under the License.
       <arg name="ErrorCode" type="CommissioningErrorEnum"/>
       <arg name="DebugText" type="char_string"/>
     </command>
+    <command source="client" code="0x06" name="SetTCAcknowledgements" optional="true" response="SetTCAcknowledgementsResponse" cli="chip fabric_commissioning settcacknowledgements" apiMaturity="provisional">
```

Comment:
The `cli` stuff is not needed and completely obsolete... You can leave it for now but please don't cut and paste things you are not sure about...